pkg/semtech-loramac/Makefile.dep

```diff
@@ -0,0 +1,6 @@
+USEMODULE += random
```

Comment:
This file has the same semantics as ```Makefile.dep``` in the root folder. So if someone does ```USEPKG += semtech-loramac```, all these modules should get selected?

Reply:
yes, do you suggest that I should add an entry in the root Makefile.dep and remove this file ?

Reply:
Looking at this file, the actual way of including dependencies seems correct